activiti-core/activiti-bpmn-converter/src/main/java/org/activiti/bpmn/converter/export/MultiInstanceExport.java

```diff
@@ -59,8 +59,11 @@ public static void writeMultiInstance(Activity activity, XMLStreamWriter xtw) th
               xtw.writeCharacters(multiInstanceObject.getCompletionCondition());
               xtw.writeEndElement();
           }
-          xtw.writeEndElement();
       }
+      else{
```

Comment:
this else statement containse the first line of the if above, so proabably we can just move out of the if. Can you also add a test? Thanks!